llvm/include/llvm/IR/IntrinsicsSPIRV.td

```diff
@@ -27,7 +27,7 @@ let TargetPrefix = "spv" in {
   def int_spv_insertv : Intrinsic<[llvm_i32_ty], [llvm_i32_ty, llvm_any_ty, llvm_vararg_ty]>;
   def int_spv_extractelt : Intrinsic<[llvm_any_ty], [llvm_any_ty, llvm_anyint_ty]>;
   def int_spv_insertelt : Intrinsic<[llvm_any_ty], [llvm_any_ty, llvm_any_ty, llvm_anyint_ty]>;
-  def int_spv_const_composite : Intrinsic<[llvm_i32_ty], [llvm_vararg_ty]>;
+  def int_spv_const_composite : Intrinsic<[llvm_any_ty], [llvm_vararg_ty]>;
```

Comment:
Please also change the description in [SPIRVUsage](https://llvm.org/docs/SPIRVUsage.html#target-intrinsics) file. It might make sense to add a comment somewhere in IntrinsicsSPIRV.td which will signal that any modifications/new intrinsics need to be also documented in SPIRVUsage.

Reply:
Indeed, thank you!